cob_description/urdf/cob4_base/base.urdf.xacro

```diff
@@ -136,8 +136,8 @@
     
     <joint name="${suffix}_rotation_joint" type="continuous">
       <axis xyz="0 0 1" />
-      <limit effort="100" velocity="100"/> <!-- alpha tested velocity and effort limits -->
-      <safety_controller  k_velocity="10" />
+      <!--limit effort="100" velocity="100"/--> <!-- alpha tested velocity and effort limits -->
+      <!--safety_controller  k_velocity="10" /-->
```

Comment:
Will this work in simulation, too?
Isn't the behavior that limits from yaml OVERWRITE urdf limits? Then we could leave the limits in here...